scripts/tasks.sh

```diff
@@ -718,6 +724,97 @@ function install_host_dependencies_longhorn() {
     longhorn_host_init_common "${DIR}/packages/host/longhorn"
 }
 
+function weave_to_flannel_primary() {
+    shift
+    while [ "$1" != "" ]; do
+        _param="$(echo "$1" | cut -d= -f1)"
+        _value="$(echo "$1" | grep '=' | cut -d= -f2-)"
+        case $_param in
+            cert-key)
+                CERT_KEY="$_value"
+                ;;
+            *)
+                echo >&2 "Error: unknown parameter \"$_param\""
```

Comment:
could we not use here logFail to ensure that will print with its colours and etc?

Reply:
Well, initially my response was "because copilot", I'll fix

but apparently copilot copied the code from `migrate_rgw_to_minio_task`, `migrate_pvcs`, and `join_token` in the same file 😆, so I'm keeping it